lib/main.go

```diff
@@ -214,7 +215,36 @@ func (obj *Main) Run() error {
 		return fmt.Errorf("hostname cannot be empty")
 	}
 
+	// Get current user running mgmt
```

Comment:
nit: remove comment b/c it's obvious or add the trailing period please. You can have full sentences (capitalized & period) or lower case mini-ones.